ds_judgements_public_ui/templates/includes/search/search_header_text.html

```diff
@@ -0,0 +1,4 @@
+<div class="search-term-component__header-beta">
+    <h2 class="search-term-component__header-text">Search for judgments and decisions from 2001 onwards</h2>
```

Comment:
In the fullness of time this probably wants to be dynamic.

Reply:
@dragon-dxw yes noted if this version is kept but if this version is discarded after testing it wont be required.